spa/config-overrides.js

```diff
@@ -0,0 +1,8 @@
+//https://stackoverflow.com/questions/70591567/module-not-found-error-cant-resolve-fs-in-react
+module.exports = function override(config, env) {
+  console.log("React app rewired works!")
+  config.resolve.fallback = {
```

Comment:
So that on github when building and package not found, it will return something instead. 
ps: funny it doesn't return `false`, it return some empty object instead.